### ISSUE TITLE
Improve the Accessibility According to WCAG and ARIA Standards 

### ISSUE THREAD

**Issue by BigSteveLittle** [RedAndBlackZone/redandblackzone#16]
This website was originally delivered as part of Certificate IV in Website Technologies that was completed in June 2021. Since then I have gone on to complete a Diploma in Website Development in December 2021.

So there are two main issues with the current site as outlined in the README.md file:

1. Major additions and re-development.
2. General fixes

This Issue is the upgrade and Improvement of the accessibility of the website according to WCAG and ARIA standards.

### Improve the Accessibility According to WCAG and ARIA Standards

- [ ] Research and document possible improvements.
- [ ] Make a list of improvements to achieve.
- [ ] Final Check and Testing.
